service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -365,14 +371,16 @@ private ApiDataReferenceDescription makeApiDataReferenceDescription(
       @PathVariable("role") ApiIamRole role,
       @RequestBody ApiGrantRoleRequestBody body) {
     ControllerValidationUtils.validateEmail(body.getMemberEmail());
+    // Sam and GCP always use lowercase email identifiers, so we do the same here for consistency.
+    String memberEmail = body.getMemberEmail().toLowerCase();
```

Comment:
I'd suggest moving the email downcase into the SamService rather than have it in the controller.